src/cli/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "abstruse-cli",
+  "version": "0.0.1",
+  "description": "Abstruse CLI",
+  "bin": {
+    "abstruse-cli": "./dist/index.js"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --config webpack.cli.js --hide-modules"
+  },
+  "keywords": [
+    "abstruse",
+    "cli"
+  ],
+  "author": "Bleenco OSS Team <oss@bleenco.com",
+  "license": "MIT",
+  "devDependencies": {
+    "node-rsa": "^0.4.2"
```

Comment:
i don't believe this is the only dependency for the `cli`. the ones I remember from head are at least `typescript`, `request`, `webpack` all loaders that `webpack` uses in configuration.

also add `chalk` and colorize outputs, it will look alot nicer.